src/shared/ml-thirdparty-simulator/index.ts

```diff
@@ -71,6 +78,12 @@ export class Simulator {
     const targetUrl = '/parties/' + type.toString() + '/' + id
     const payload = mockPutPartiesRequest(type, id)
 
+    if (this.opts.delay) {
+      // Delay operations to simulate network latency in real communication
+      // with Mojaloop.
+      await this.delay(1500)
+    }
```

Comment:
what about delay 2000 defined here: src/server/plugins/index.ts
maybe it would be worth to use global config for this magic number in source code?

Reply:
Oh right, forgot to change this before, updated :)